chef/cookbooks/tacker/templates/default/tacker.conf.erb

```diff
@@ -0,0 +1,57 @@
+[DEFAULT]
+lock_path = $state_path/lock
+bind_host = <%= @bind_host %>
+bind_port = <%= @bind_port %>
+service_plugins = tacker.vm.plugin.VNFMPlugin,tacker.sfc.plugin.SFCPlugin,tacker.sfc_classifier.plugin.SFCCPlugin 
+control_exchange = tacker
+rabbit_password = <%= @rabbit_settings[:password] %>
+rabbit_hosts = <%= @rabbit_settings[:address] %>
+rabbit_userid = <%= @rabbit_settings[:user] %>
+rabbit_virtual_host = <%= @rabbit_settings[:vhost] %>
+rabbit_ha_queues = True
+notification_driver = tacker.openstack.common.notifier.rpc_notifier
+
+[agent]
+
+[keystone_authtoken]
+signing_dir = /var/cache/tacker
+cafile = /opt/stack/data/ca-bundle.pem
+project_domain_id = default
+project_name = <%= @keystone_settings["service_tenant"] %>
+user_domain_id = default
+password = <%= @keystone_settings["service_password"] %>
+username = <%= @keystone_settings["service_user"] %>
+auth_url = <%= @keystone_settings["admin_auth_url"] %>
+auth_plugin = password
+identity_uri = <%= @keystone_settings["admin_auth_url"] %>
+auth_uri = <%= @keystone_settings["internal_auth_url"] %>
+
+[database]
+connection = <%= @connection %> 
+max_retries = 10
+retry_interval = 10
+min_pool_size = 1
+max_pool_size = 10
+idle_timeout = 3600
+max_overflow = 20
+
+[servicevm]
+infra_driver = heat
+mgmt_driver = noop
+mgmt_driver = openwrt
+
+[servicevm_heat]
+heat_uri = <%= @heat_uri %>
+stack_retries = 10
+stack_retry_wait = 30
+
+[servicevm_agent]
+
+[sfc]
+infra_driver = opendaylight
+
+[sfc_opendaylight]
```

Comment:
Would it help to use variables for IP and port here ? Because,the ODL deployment may change.